WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -283,6 +283,14 @@ - (NSData *)fb_screenshotWithError:(NSError **)error
   }
 
   CGRect elementRect = self.frame;
+
+  if (@available(iOS 13.0, *)) {
```

Comment:
As https://github.com/appium/WebDriverAgent/pull/231 , taking the element screenshot also works without orientation calculation.